docs/en/latest/deployments/kubesphere.md

```diff
@@ -52,6 +52,25 @@ helm install apisix apisix/apisix \
 kubectl get service --namespace ingress-apisix
 ```
 
+:::tip
+
+If you have problems deploying APISIX due to network reasons, you can download, copy, and install APISIX as shown below:
+
+```shell
+VERSION=$(curl https://artifacthub.io/api/v1/packages/helm/apisix/apisix | jq -r ".version")
```

Comment:
I am wondering if this link is permanent. Can we use GitHub API?
For example https://api.github.com/repos/apache/apisix-ingress-controller/tags